### ISSUE TITLE
Data visualization

### ISSUE THREAD

**Issue by MiguelSteph** [MiguelSteph/heart_disease_prediction#8]
Use PCA to separately visualize the training set, the validation set and the test set
